cluster/common.sh

```diff
@@ -619,6 +619,8 @@ ENABLE_L7_LOADBALANCING: $(yaml-quote ${ENABLE_L7_LOADBALANCING:-none})
 ENABLE_CLUSTER_LOGGING: $(yaml-quote ${ENABLE_CLUSTER_LOGGING:-false})
 ENABLE_CLUSTER_UI: $(yaml-quote ${ENABLE_CLUSTER_UI:-false})
 ENABLE_NODE_PROBLEM_DETECTOR: $(yaml-quote ${ENABLE_NODE_PROBLEM_DETECTOR:-none})
+NPD_VERSION: $(yaml-quote ${NPD_VERSION:-})
```

Comment:
Can we use `NODE_PROBLEM_DETECTOR` instead of `NPD` to make it more clear and also make it consistent with `ENABLE_NODE_RPBOLEM_DETECTOR` above?